src/core/server/http/http_server.mocks.ts

```diff
@@ -42,9 +42,14 @@ interface RequestFixtureOptions {
   method?: RouteMethod;
   socket?: Socket;
   routeTags?: string[];
+  validation?: {
+    params?: Type<P>;
```

Comment:
The same PR allows plugins to define validation via custom validators. I suspect we shouldn't use `@kbn/config-schema` specific type here. WDYT?

Reply:
Good point, I will use the generic interface 👍